xml/System.Text/Rune.xml

```diff
@@ -1694,10 +1694,20 @@ For more information, see <xref:System.Char.IsWhiteSpace%2A?displayProperty=name
         <Parameter Name="obj" Type="System.Object" Index="0" FrameworkAlternate="net-5.0" />
       </Parameters>
       <Docs>
-        <param name="obj">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="other">The object to compare with the current instance.</param>
+        <summary>Compares the current instance to the specified object.</summary>
+        <returns>A signed integer indicating the position of this instance in the sort order in relation to <paramref name="other" />: <br />- Less than zero: This instance precedes <paramref name="other" />. 
```

Comment:
Maybe consider re-wording this similar to https://docs.microsoft.com/en-us/dotnet/api/system.single.compareto?view=netcore-3.1#System_Single_CompareTo_System_Object_ ?